### ISSUE TITLE
Pasting on Windows adds text `EndFragment`

### ISSUE THREAD

**Issue by pablopunk** [aem/docs-soap#35]
**Provide a detailed description of the issue**

Whenever I use Chrome on Windows and I paste some simple text from google docs using docs-soap, I always get some extra text at the end `EndFragment`

**Steps to reproduce**

1. Copy text from Gdocs
2. Paste it into draft-js using docs-soap to handle the HTML
3. At least using Chrome on Windows, the text `EndFragment` is added to the pasted chunk

**Expected behavior**

No text should be added

**Actual behavior**

Text is added

**Code example or screenshot if applicable**

See this video I made https://drive.google.com/file/d/1TrBHr3ylyDz2nyExXgW3KzeTnQJT-jPe/view

@aem

**Comment by pablopunk** [comment]
I took a look at the html before and after using docs-soap and it seems the library is converting the last comment in text:

### Before

```html
<!--StartFragment-->
  <meta charset="utf-8">
  <b style="font-weight:normal;" id="docs-internal-guid-70b18ce0-8c93-b02f-859e-fed642087eb7"> 
    <span style="font-size:12pt;font-family:Roboto;color:#333333;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">
      Pasting something from google doc</span>
  </b>
<!--EndFragment-->
```

### After

```html
 <body><span>Pasting something from google doc</span><span>EndFragment</span><span>
```

**Comment by pablopunk** [comment]
FYI I'm working on a PR to remove comments from output

**Comment by aem** [comment]
Good to hear on the PR. I’m mobile now, on vacation in Europe until June 1,
but I’ll take a look when I’m back stateside.

**Comment by pablopunk** [comment]
@aem I recreated the issue with a test and made the PR already, is there anyway I can get this merged soon? We really need it

Thanks for the quick response BTW

**Comment by aem** [comment]
fixed in 1.2.1
